pkg/protocol/engine/mempool/errors.go

```diff
@@ -1,5 +1,10 @@
 package mempool
 
-import "github.com/iotaledger/hive.go/ierrors"
+import (
+	"github.com/iotaledger/hive.go/ierrors"
+)
 
-var ErrStateNotFound = ierrors.New("state not found")
+var (
+	ErrStateNotFound = ierrors.New("state not found")
+	ErrRequestFailed = ierrors.New("request failed")
```

Comment:
Being more specific here would be great, as "request failed" is very generic. Since we use it only in one place where the error is about solidification of inputs, perhaps:

```suggestion
	ErrInputSolidificationRequestFailed = ierrors.New("UTXO input solidification failed")
```